src/test/modelUtils.ts

```diff
@@ -41,3 +41,7 @@ export function createWorkflowName(
 ) {
   return createNamedEntity(ResourceType.WORKFLOW, id, metadata);
 }
+
+export function createTask(id: NamedEntityIdentifier, metadata?: Partial<NamedEntityMetadata>) {
```

Comment:
Looks like `createTask` is used only in `src/components/Project/test/ProjectTask.test.tsx`, as such it is better to declare it in test file, to avoid unneeded exports.
Maybe you can also move `createWorkflowName` into `src/components/Project/test/ProjectWorkflows.test.tsx` and remove from here too, as it should be a quick and non-invasive change.